Doc/library/typing.rst

```diff
@@ -2096,15 +2096,15 @@ Corresponding to collections in :mod:`collections.abc`
       :class:`collections.abc.Container` now supports ``[]``. See :pep:`585`
       and :ref:`types-genericalias`.
 
-.. class:: ItemsView(MappingView, Generic[KT_co, VT_co])
+.. class:: ItemsView(MappingView, AbstractSet[tuple[KT_co, VT_co]], Generic[KT_co, VT_co])
```

Comment:
Generic is redundant as a base class, because there is a generic base. Other classes nearby don't list Generic as an explicit base, so these ones shouldn't either.

Reply:
Yes, I feel like the fact that so many typeshed classes do this in the stubs is something of a legacy thing (though some may prefer it for added explicitness).

Reply:
Done!